docs/_data/sidebar-core.yml

```diff
@@ -3,139 +3,62 @@ options:
 
     nested_options:
 
-      - title: Index
-        url: /core/
-
       - title: Setup
         url: /quickstart/setup/
 
-      - title: Linting
-        url: /quickstart/linting/
-
-      - title: Libraries
-        url: /quickstart/libraries/
-
       - title: Media
-        url: https://media.arrow-kt.io
+        url: https://media.arrow-kt.io/tags/optics/
```

Comment:
Why `/tags/optics`?

Reply:
ops, wrong copy/pasta. This was supposed to be in the Media link in the optics site, not core.

Reply:
should be fixed now, each lib pointing to their media tag